### PR TITLE
fixed constructor link descriptions 4

### PR DESCRIPTION
**Pull request by OnkarRuikar**
#### Summary
Related to #14001

Fixed syntax section of constructor pages. Removed variables and square brackets.

#### Motivation
Following the guidelines:
https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections#constructors_and_methods

#### Metadata

- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/animationevent/animationevent/index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The **`AnimationEvent()`** constructor returns a newly created
 ## Syntax
 
 ```js
-animationEvent = new AnimationEvent(type, {animationName: aPropertyName,
+new AnimationEvent(type, {animationName: aPropertyName,
                                            elapsedTime  : aFloat,
                                            pseudoElement: aPseudoElementName});
```
Comment (teoli2003) [inline files/en-us/web/api/animationevent/animationevent/index.md:23]: We should align `animationName`, `elapseTime` and `pseudoElement`.

File: files/en-us/web/api/blobevent/blobevent/index.md [hunk 2]
```diff
@@ -20,7 +20,8 @@ The **`BlobEvent()`** constructor returns a newly created
 ## Syntax
 
 ```js
-blobEvent = new BlobEvent({data: aSpecificBlob}[, timecode]);
+ new BlobEvent({data: aSpecificBlob});
+ new BlobEvent({data: aSpecificBlob}, timecode);
```
Comment (teoli2003) [inline files/en-us/web/api/blobevent/blobevent/index.md:24]:
```suggestion
new BlobEvent({data: aSpecificBlob});
new BlobEvent({data: aSpecificBlob}, timecode);
```

File: files/en-us/web/api/broadcastchannel/broadcastchannel/index.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ The **`BroadcastChannel()`** constructor creates a new
 ## Syntax
 
 ```js
- channel = new BroadcastChannel(channelName);
+ new BroadcastChannel(channelName);
```
Comment (teoli2003) [inline files/en-us/web/api/broadcastchannel/broadcastchannel/index.md:23]:
```suggestion
new BroadcastChannel(channelName);
```

File: files/en-us/web/api/constantsourcenode/constantsourcenode/index.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ constantly outputs samples whose values are always the same.
 ## Syntax
 
 ```js
-var constantSourceNode = new ConstantSourceNode(context, options);
+ new ConstantSourceNode(context, options);
```
Comment (teoli2003) [inline files/en-us/web/api/constantsourcenode/constantsourcenode/index.md:23]:
```suggestion
new ConstantSourceNode(context, options);
```

File: files/en-us/web/api/cssmatrixcomponent/cssmatrixcomponent/index.md [hunk 5]
```diff
@@ -21,7 +21,8 @@ individual {{CSSXRef('transform')}} property in CSS.
 ## Syntax
 
 ```js
-new CSSMatrixComponent(matrix[,options]);
+ new CSSMatrixComponent(matrix);
+ new CSSMatrixComponent(matrix, options);
```
Comment (teoli2003) [inline files/en-us/web/api/cssmatrixcomponent/cssmatrixcomponent/index.md:25]:
```suggestion
new CSSMatrixComponent(matrix);
new CSSMatrixComponent(matrix, options);
```

File: files/en-us/web/api/cssscale/cssscale/index.md [hunk 6]
```diff
@@ -21,7 +21,8 @@ individual {{CSSXref('transform')}} property in CSS.
 ## Syntax
 
 ```js
-new CSSScale(x,y[,z]);
+ new CSSScale(x, y);
+ new CSSScale(x, y, z);
```
Comment (teoli2003) [inline files/en-us/web/api/cssscale/cssscale/index.md:25]:
```suggestion
new CSSScale(x, y);
new CSSScale(x, y, z);
```

File: files/en-us/web/api/csstranslate/csstranslate/index.md [hunk 7]
```diff
@@ -21,7 +21,8 @@ individual {{CSSXref('transform')}} property in CSS.
 ## Syntax
 
 ```js
-new CSSTranslate(x,y[,z]);
+ new CSSTranslate(x, y);
+ new CSSTranslate(x, y, z);
```
Comment (teoli2003) [inline files/en-us/web/api/csstranslate/csstranslate/index.md:25]:
```suggestion
new CSSTranslate(x, y);
new CSSTranslate(x, y, z);
```

File: files/en-us/web/api/cssvariablereferencevalue/cssvariablereferencevalue/index.md [hunk 8]
```diff
@@ -18,7 +18,8 @@ Creates a new {{domxref('CSSVariableReferenceValue')}}.
 ## Syntax
 
 ```js
-new CSSVariableReferenceValue(variable[, fallback]])
+ new CSSVariableReferenceValue(variable)
+ new CSSVariableReferenceValue(variable, fallback)
```
Comment (teoli2003) [inline files/en-us/web/api/cssvariablereferencevalue/cssvariablereferencevalue/index.md:22]:
```suggestion
new CSSVariableReferenceValue(variable)
new CSSVariableReferenceValue(variable, fallback)
```

File: files/en-us/web/api/file/file/index.md [hunk 9]
```diff
@@ -16,7 +16,8 @@ object instance.
 ## Syntax
 
 ```js
-new File(bits, name[, options]);
+ new File(bits, name);
+ new File(bits, name, options);
```
Comment (teoli2003) [inline files/en-us/web/api/file/file/index.md:20]:
```suggestion
new File(bits, name);
new File(bits, name, options);
```

File: files/en-us/web/api/formdataevent/formdataevent/index.md [hunk 10]
```diff
@@ -18,7 +18,8 @@ The **`FormDataEvent()`** constructor creates a new
 ## Syntax
 
 ```js
-new FormDataEvent(type[, formEventInit]);
+ new FormDataEvent(type);
+ new FormDataEvent(type, formEventInit);
 ```
```
Comment (teoli2003) [inline files/en-us/web/api/formdataevent/formdataevent/index.md:23]:
```suggestion
new FormDataEvent(type);
new FormDataEvent(type, formEventInit);
```
```

File: files/en-us/web/api/keyboardevent/keyboardevent/index.md [hunk 11]
```diff
@@ -17,7 +17,7 @@ The **`KeyboardEvent()`** constructor creates a new
 ## Syntax
 
 ```js
- event = new KeyboardEvent(typeArg, KeyboardEventInit);
+ new KeyboardEvent(typeArg, KeyboardEventInit);
```
Comment (teoli2003) [inline files/en-us/web/api/keyboardevent/keyboardevent/index.md:20]:
```suggestion
new KeyboardEvent(typeArg, KeyboardEventInit);
```

File: files/en-us/web/api/mediastreamaudiosourcenode/mediastreamaudiosourcenode/index.md [hunk 12]
```diff
@@ -25,7 +25,7 @@ the first audio track of a given {{domxref("MediaStream")}} as its source.
 ## Syntax
 
 ```js
-audioSourceNode = new MediaStreamAudioSourceNode(context, options);
+ new MediaStreamAudioSourceNode(context, options);
```
Comment (teoli2003) [inline files/en-us/web/api/mediastreamaudiosourcenode/mediastreamaudiosourcenode/index.md:28]:
```suggestion
new MediaStreamAudioSourceNode(context, options);
```

File: files/en-us/web/api/merchantvalidationevent/merchantvalidationevent/index.md [hunk 13]
```diff
@@ -21,7 +21,7 @@ The **`MerchantValidationEvent()`** constructor creates a new {{domxref("Merchan
 ## Syntax
 
 ```js
-merchantValidationEvent = new MerchantValidationEvent(type, options);
+ new MerchantValidationEvent(type, options);
```
Comment (teoli2003) [inline files/en-us/web/api/merchantvalidationevent/merchantvalidationevent/index.md:24]:
```suggestion
new MerchantValidationEvent(type, options);
```

File: files/en-us/web/api/mouseevent/mouseevent/index.md [hunk 14]
```diff
@@ -18,7 +18,7 @@ The **`MouseEvent()`** constructor creates a new
 ## Syntax
 
 ```js
- event = new MouseEvent(typeArg, mouseEventInit);
+ new MouseEvent(typeArg, mouseEventInit);
```
Comment (teoli2003) [inline files/en-us/web/api/mouseevent/mouseevent/index.md:21]:
```suggestion
new MouseEvent(typeArg, mouseEventInit);
```

File: files/en-us/web/api/paymentrequest/paymentrequest/index.md [hunk 15]
```diff
@@ -21,7 +21,8 @@ process of generating, validating, and submitting a payment request.
 ## Syntax
 
 ```js
-var paymentRequest = new PaymentRequest(methodData, details, [options]);
+ new PaymentRequest(methodData, details);
+ new PaymentRequest(methodData, details, options);
```
Comment (teoli2003) [inline files/en-us/web/api/paymentrequest/paymentrequest/index.md:25]:
```suggestion
new PaymentRequest(methodData, details);
new PaymentRequest(methodData, details, options);
```

File: files/en-us/web/api/pointerevent/pointerevent/index.md [hunk 16]
```diff
@@ -16,7 +16,7 @@ and untrusted {{domxref("PointerEvent")}} object instance.
 ## Syntax
 
 ```js
- event = new PointerEvent(type, PointerEventInit);
+ new PointerEvent(type, PointerEventInit);
```
Comment (teoli2003) [inline files/en-us/web/api/pointerevent/pointerevent/index.md:19]:
```suggestion
new PointerEvent(type, PointerEventInit);
```

File: files/en-us/web/api/progressevent/progressevent/index.md [hunk 17]
```diff
@@ -16,7 +16,7 @@ The **`ProgressEvent()`** constructor returns a newly created
 ## Syntax
 
 ```js
-progressEvent = new ProgressEvent(type, {lengthComputable: aBooleanValue, loaded: aNumber, total: aNumber});
+ new ProgressEvent(type, {lengthComputable: aBooleanValue, loaded: aNumber, total: aNumber});
```
Comment (teoli2003) [inline files/en-us/web/api/progressevent/progressevent/index.md:19]:
```suggestion
new ProgressEvent(type, {lengthComputable: aBooleanValue, loaded: aNumber, total: aNumber});
```

File: files/en-us/web/api/reportingobserver/reportingobserver/index.md [hunk 18]
```diff
@@ -19,7 +19,8 @@ access reports.
 ## Syntax
 
 ```js
-new ReportingObserver(callback[, options]);
+ new ReportingObserver(callback);
+ new ReportingObserver(callback, options);
```
Comment (teoli2003) [inline files/en-us/web/api/reportingobserver/reportingobserver/index.md:23]:
```suggestion
new ReportingObserver(callback);
new ReportingObserver(callback, options);
```

File: files/en-us/web/api/rtcsessiondescription/rtcsessiondescription/index.md [hunk 19]
```diff
@@ -28,7 +28,7 @@ specified object.
 ## Syntax
 
 ```js
-sessionDescription = new RTCSessionDescription(rtcSessionDescriptionInit);
+ new RTCSessionDescription(rtcSessionDescriptionInit);
```
Comment (teoli2003) [inline files/en-us/web/api/rtcsessiondescription/rtcsessiondescription/index.md:31]:
```suggestion
new RTCSessionDescription(rtcSessionDescriptionInit);
```

File: files/en-us/web/api/sensorerrorevent/sensorerrorevent/index.md [hunk 20]
```diff
@@ -22,7 +22,8 @@ errors thrown by any of the interfaces based on {{domxref('Sensor')}}.
 ## Syntax
 
 ```js
-sensorErrorEvent = new SensorErrorEvent(type, {error: domException});
+ new SensorErrorEvent(type);
+ new SensorErrorEvent(type, {error: domException});
```
Comment (teoli2003) [inline files/en-us/web/api/sensorerrorevent/sensorerrorevent/index.md:26]:
```suggestion
new SensorErrorEvent(type);
new SensorErrorEvent(type, {error: domException});
```

File: files/en-us/web/api/textdecoder/textdecoder/index.md [hunk 21]
```diff
@@ -22,7 +22,7 @@ If the value for _utfLabel_ is unknown, or is one of the two values leading to a
 ## Syntax
 
 ```js
-decoder = new TextDecoder(utfLabel, options);
+ new TextDecoder(utfLabel, options);
```
Comment (teoli2003) [inline files/en-us/web/api/textdecoder/textdecoder/index.md:25]:
```suggestion
new TextDecoder(utfLabel, options);
```

File: files/en-us/web/api/transitionevent/transitionevent/index.md [hunk 22]
```diff
@@ -20,7 +20,7 @@ The **`TransitionEvent()`** constructor returns a newly created
 ## Syntax
 
 ```js
-transitionEvent = new TransitionEvent(type, {propertyName: aPropertyName,
+new TransitionEvent(type, {propertyName: aPropertyName,
                                              elapsedTime  : aFloat,
                                              pseudoElement: aPseudoElementName});
```
Comment (teoli2003) [inline files/en-us/web/api/transitionevent/transitionevent/index.md:25]: I think we should align the three `propertyName`, `elapsedTime`, and `pseudoElement`.

File: files/en-us/web/api/uievent/uievent/index.md [hunk 23]
```diff
@@ -16,7 +16,8 @@ The **`UIEvent()`** constructor creates a new
 ## Syntax
 
 ```js
-event = new UIEvent(typeArg [, UIEventInit])
+ new UIEvent(typeArg);
+ new UIEvent(typeArg, UIEventInit);
```
Comment (teoli2003) [inline files/en-us/web/api/uievent/uievent/index.md:20]:
```suggestion
new UIEvent(typeArg);
new UIEvent(typeArg, UIEventInit);
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
A few alignments to fix.

File: files/en-us/web/api/animationevent/animationevent/index.md (hunk 1 above)
Comment (OnkarRuikar) [reply files/en-us/web/api/animationevent/animationevent/index.md]: done

File: files/en-us/web/api/transitionevent/transitionevent/index.md (hunk 22 above)
Comment (teoli2003) [reply files/en-us/web/api/transitionevent/transitionevent/index.md:25]: @OnkarRuikar, I think this one is still unresolved.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Comment by OnkarRuikar** [comment]
@teoli2003  done.

All constructor pages are done now. 1088 properties pages to go.

File: files/en-us/web/api/transitionevent/transitionevent/index.md (hunk 22 above)
Comment (OnkarRuikar) [reply files/en-us/web/api/transitionevent/transitionevent/index.md]: ohh there were two of them. Fixed it.

**Review by teoli2003 — APPROVED** [review]
Perfect!

**Pull request merged**